.gitignore

```diff
@@ -20,6 +20,7 @@ __docs-temp__/*
 *.bak
 *.orig
 .stencil/
+.storybook/
```

Comment:
This shouldn't be ignored.

Reply:
@jcfranco should we check in these files then?